fdb-record-layer-spatial/fdb-record-layer-spatial.gradle

```diff
@@ -80,8 +67,9 @@ task unzipGeonames(dependsOn: downloadGeonames, type: Copy) {
     }
 }
 
+def skipSlow = System.getenv('SKIP_SLOW_TESTS') != null && System.getenv('SKIP_SLOW_TESTS') == 'true'
 if (!skipSlow) {
-    test.dependsOn unzipGeonames
+    processTestResources.dependsOn unzipGeonames
```

Comment:
Gradle started popping up warnings that `processTestResources` depended on the output of `unzipGeonames`, but it wasn't explicit, and that could cause problems.